system/include/webgl/webgl1_ext.h

```diff
@@ -5,18 +5,32 @@
 
 // 1. https://www.khronos.org/registry/webgl/extensions/OES_texture_float/
 // <no symbols exposed>
+#ifndef EMSCRIPTEN_OES_texture_float
+#define EMSCRIPTEN_OES_texture_float 1
```

Comment:
nit: Maybe move the `// <no symbols exposed>` to here, where the symbols would be if there were any